sormas-rest/src/main/java/de/symeda/sormas/rest/TravelEntryResource.java

```diff
@@ -0,0 +1,65 @@
+package de.symeda.sormas.rest;
+
+import java.util.List;
+
+import javax.annotation.security.RolesAllowed;
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import de.symeda.sormas.api.FacadeProvider;
+import de.symeda.sormas.api.PushResult;
+import de.symeda.sormas.api.caze.CriteriaWithSorting;
+import de.symeda.sormas.api.common.Page;
+import de.symeda.sormas.api.travelentry.TravelEntryCriteria;
+import de.symeda.sormas.api.travelentry.TravelEntryDto;
+import de.symeda.sormas.api.travelentry.TravelEntryIndexDto;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+
+@Path("/travelEntry")
```

Comment:
Please stick to the naming conventions of other resources; i.e. plural, everything written in lowercase (=> travelentries)